packages/carbon-react/docs/sass.md

```diff
@@ -0,0 +1,85 @@
+# Sass
+
+<!-- prettier-ignore-start -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [Overview](#overview)
+- [Configuration](#configuration)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- prettier-ignore-end -->
+
+## Overview
+
+The Carbon Design System requires [Dart Sass](http://npmjs.com/package/sass) to
+compile the styles for components. 
+
+## Configuration
+
+You will need to configure Sass to be able to lookup packages from your
+`node_modules` folder. To do this, use the `includePaths` option and set its
+value to an array of locations where Sass should look to find `node_modules`
+folders.
+
+For most teams, this configuration will look like:
+
+```json
+{
+  "includePaths": ["node_modules"]
+}
+```
+
+For bundler specific solutions, check out the sections below for your bundler
+of choice. If you can't find what you're looking for, please make an
+[issue](https://github.com/carbon-design-system/carbon/issues/new/choose) and
+we'll try to get instructions for it added!
+
+**`sass-loader`**
+
+[Link](https://www.npmjs.com/package/sass-loader)
+
+Update your `webpack.config.js` that uses `sass-loader` with the following
+options passed into `sassOptions`:
+
+```js
+{
+  loader: 'sass-loader',
+  options: {
+    sassOptions: {
+      includePaths: ['node_modules'],
+    },
+  },
```

Comment:
I know `sass-loader` prefers the dart sass implementation by default, but should we call that out here? and or just recommend configuring the `implementation` option?

```suggestion
  options: {
    implementation: require("sass"),
    sassOptions: {
      includePaths: ['node_modules'],
    },
  },
```

Reply:
Maybe we could add a note like "if the styles aren't working, you might be using node-sass in your project" and then use this as a way to tell if that's the case?